src/components/Button/Button.tsx

```diff
@@ -35,17 +35,22 @@ class Button extends UIComponent<any, any> {
 
     /** A button can be formatted to show different levels of emphasis. */
     type: PropTypes.oneOf(['primary', 'secondary']),
+
+    a11yType: PropTypes.string,
```

Comment:
I would change the property name to something like ``acc-behavior``

Reply:
ok, will change